lambda/Scheduler/index.js

```diff
@@ -0,0 +1,36 @@
+'use strict'
+
+process.env['PATH'] = `${process.env['PATH']}:${process.env['LAMBDA_TASK_ROOT']}`;
+
+let config = require('./config.json');
+let schedulerFactory = require('./scheduler.js');
+
+exports.handler = (event, context, callback) => {
+
+  let scheduler = schedulerFactory.create(config);
+
+  try {
+
+    scheduler.doScheduling()
+    .then(result => {
+      if (result.success)
```

Comment:
could you use block scope { }, as in AirBnb styleguide? https://github.com/airbnb/javascript#blocks--braces